core/src/main/java/org/frankframework/core/ISender.java

```diff
@@ -51,6 +51,7 @@ public interface ISender extends IConfigurable {
 	/**
 	 * Stop/close the sender and deallocate resources.
 	 */
+	@Override
 	void close() throws SenderException;
```

Comment:
Ik heb ooit https://github.com/frankframework/frankframework/issues/5045 aangemaakt om open/close te vervangen door start en stop. Mocht een pipe/sender dan zelf toch nog AutoCloseable willen implementeren dan kan dat maar het moet niet een verplichting zijn.

Reply:
En wil je deze AutoClosable nog weghalen of...?

Reply:
Ik vind eigenlijk de AutoClosable of een sender nog steeds wel logisch, maar nu weggehaald, zie commit:
https://github.com/frankframework/frankframework/pull/6740/commits/cbd3334858cbc2dc25827e8ab169d5b62b9ed120